.github/MAINTAINER_GUIDELINES.md

```diff
@@ -0,0 +1,104 @@
+
+## General expectations
+
+- Self assign issues/pull requests (mostly pull requests)
+  - Review and approve or request changes
+  - Enforce the use of the issue/pull requests template
+  - Ask contributors to link to the code base to validate documentation updates when applicable
+  - Ask contributors not to contribute to v2.0 docs
+- If a maintainer creates a pull request, it should be reviewed by another maintainer or DevDocs staff member
+
+## Labels
+
+Labels are important because they help us identify pull requests and ensure that contributors receive points and recognition. See [devdocs awards and points](https://devdocs.magento.com/guides/v2.3/contributor-guide/contributing.html#devdocs-awards-and-points).
+
+Here is a brief summary of the most important labels:
+
+- `New Topic`: Entirely new documents
+- `Major Update`: Significant new info: new section in existing topic, etc.
+- `Technical`: Changes to technical content/code/processes/naming conventions (any change to technical content)
+- `Editorial`: Typos, grammatical inconsistencies, or minor rewrites
+- `small changes`: See [Small changes workflow](#small-changes-workflow)
+
+We also use version labels when appropriate.
+
+As a maintainer, we expect you to add or remove labels when appropriate.
+
+See https://github.com/magento/devdocs/labels for all labels and their descriptions.
+
+## Testing
+
+Currently, we only test internal links. In the future, we plan to expand tests to include markdown linting and spell checking.
+
+We use Jenkins to build, test, and deploy the devdocs site. Unfortunately, our Jenkins host is behind the corporate firewall (per Adobe security policy) and is not accessible to non-employees.
+
+Every pull request to the `master` branch must pass tests before it can be merged. When a pull request is ready for tests, a member of the [`devdocs-admins`](https://github.com/orgs/magento/teams/devdocs-admins) team must add the test trigger phrase to the pull request as a comment. The trigger phrase is "_running tests_". By approving a pull request, you are signalling an admin that the pull request is ready for tests.
+
+## Projects
+
+We use several projects to help organize issues and pull requests. You can add existing issues and pull requests to these projects as you see fit.
+
+https://github.com/magento/devdocs/projects
+
+## Style
+
+We plan to explain style in depth in the repository wiki soon, but for now, use the following guidance on markdown syntax:
+
+- Pull requests should contain simple, clean markdown syntax and not HTML (in most cases)
+- We use non-standard syntax in some places for templating (liquid) and accessing Jekyll variables
+- We sometimes use HTML for _complex_ tables, but we prefer markdown formatting in most cases
+
+## Small changes workflow
```

Comment:
The _Small changes_ is the largest section in this document.
I think maintainers don't need all the CI/CD details. We can just say that instead of processing 10 tiny PRs on our CI/CD (that takes quite a bit of time), we are processing only one PR that contains all those small changes. This releases for us more time to process larger pull requests.

Reply:
I'll simplify it.